src/Jellyfin.Extensions/StringExtensions.cs

```diff
@@ -9,8 +9,21 @@ namespace Jellyfin.Extensions
     /// </summary>
     public static partial class StringExtensions
     {
-        private static readonly Lazy<Transliterator> _transliterator = new(() => Transliterator.GetInstance(
-            "Any-Latin; Latin-Ascii; Lower; NFD; [:Nonspacing Mark:] Remove; [:Punctuation:] Remove;"));
+        private static readonly Lazy<string> _transliteratorId = new(() =>
+            Environment.GetEnvironmentVariable("JELLYFIN_TRANSLITERATOR_ID")
```

Comment:
I'm personally fine with reading env var directly but I'm not sure if there is any better way to integrate the configuration for string extensions